pyirf/tests/test_integration.py

```diff
@@ -0,0 +1,50 @@
+import glob
+import subprocess
+import logging
+import os
+from astropy.io import fits
+from ogadf_schema.irfs import AEFF_2D, EDISP_2D, PSF_TABLE, BKG_2D, RAD_MAX
+
+
+def test_integration_test(caplog):
+
+    caplog.set_level(logging.INFO)
+
+    ROOT_DIR = os.path.dirname(os.path.abspath("setup.py"))
```

Comment:
This is not helpful as it will also only work in the root. It is basically the same as using `os.getcwd` since you use a relative path for `setup.py`. `os.abspath` will not look for a `setup.py`, it will just add the current working directory in front of the given path.

You can access the absolute path of the current file using the global variable `__file__` and move from there.

E.g. `ROOT_DIR = os.path.join(__file__, '../..')`